test/Chirp.Razor.Tests/ChirpTests.cs

```diff
@@ -0,0 +1,102 @@
+using Chirp.Core.Classes;
+using Chirp.Core.Interfaces;
+using Chirp.Razor.Services;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.VisualStudio.TestPlatform.TestHost;
+
+namespace Chirp.Razor.Tests;
+
+public class UnitTests
+{
+    [Fact]
+    public void TestGetCheeps()
+    {
+        // arrange
+        ICheepService CheepService = new CheepService();
+
+        // act
+        var cheeps = CheepService.GetCheeps();
+
+        // assert
+        // 
+        Assert.NotEmpty(cheeps);
```

Comment:
Is it the right criteria that the list is not empty? What if there just are no cheeps in the db?

Reply:
does [aa76d28](https://github.com/ITU-BDSA2024-GROUP9/Chirp/pull/22/commits/aa76d284ac02da7620ced9ba868bdbc706696e46) fix this?

Reply:
Yes!